README.md

```diff
@@ -37,7 +37,7 @@ Add this dependency to your project's POM:
 <dependency>
   <groupId>com.adyen</groupId>
   <artifactId>adyen-java-api-library</artifactId>
-  <version>2.4.1</version>
+  <version>2.5.0</version>
```

Comment:
We increase the version number when merging develop to master. Could you please revert it ?